src/Core/src/Layouts/VerticalStackLayoutManager.cs

```diff
@@ -41,17 +40,16 @@ static Size Measure(double widthConstraint, int spacing, IReadOnlyList<IView> vi
 			return new Size(requestedWidth, totalRequestedHeight);
 		}
 
-		static void Arrange(int spacing, IEnumerable<IView> views)
+		static void Arrange(double width, int spacing, IEnumerable<IView> views)
 		{
 			double stackHeight = 0;
 
 			foreach (var child in views)
 			{
-				var destination = new Rectangle(0, stackHeight, child.DesiredSize.Width, child.DesiredSize.Height);
+				var destination = new Rectangle(0, stackHeight, width, child.DesiredSize.Height);
```

Comment:
Is this a "force the child to be the width" or is it just the value that is suggested to the child and the child can use a narrower width?

Reply:
The latter. The stack layout is arranging the child within that area; the child itself determines what its final Frame is. When determining its Frame, it uses the LayoutAlignment. If the alignment is Fill, it uses all the space; if Start|End|Center, it uses its DesiredSize and sets the X,Y appropriately.